lib/web_ui/lib/src/engine/js_interop/js_loader.dart

```diff
@@ -44,34 +53,34 @@ extension FlutterLoaderExtension on FlutterLoader {
 @anonymous
 @staticInterop
 abstract class FlutterEngineInitializer{
-  external factory FlutterEngineInitializer({
+  factory FlutterEngineInitializer({
     required InitializeEngineFn initializeEngine,
     required ImmediateRunAppFn autoStart,
+  }) => FlutterEngineInitializer._(
+      initializeEngine: (() => futureToPromise(initializeEngine())).toJS,
```

Comment:
Is this swallowing the parameters passed to the initializeEngine fn from JS?

I suspect this needs to be:

```
initializeEngine: (([JsFlutterConfiguration?] args) => futureToPromise(initializeEngine(args))).toJS,
```

(testing)

Reply:
Oh dang, you're right. Let me fix that.